packages/cli/src/cli-default-servient.ts

```diff
@@ -56,7 +56,9 @@ export default class DefaultServient extends Servient {
         super();
 
         // init config
-        this.config = (typeof config === "object") ? config : DefaultServient.defaultConfig;
+        this.config = (typeof config === "object") ?
+            mergeConfigs(DefaultServient.defaultConfig, config) :
+            DefaultServient.defaultConfig;
         if (!this.config.servient) this.config.servient = DefaultServient.defaultConfig.servient;
```

Comment:
I believe the line about `this.config.servient` can be removed since it should be handled in the line before. Am I wrong?

Reply:
No, you are right. I have overlooked that, I'd change the PR now.
Thank you!